### ISSUE TITLE
Enabling GPU on Sagemaker for TextPredictor (multimodal)

### ISSUE THREAD

**Issue by KaBar1985** [awslabs/autogluon#1430]
Hi,
I'm using `ml.g4dn.2xlarge` instance on SageMaker, with `conda_mxnet_p36` kernel, and would like to run : 

`predictor = TabularPredictor(label="detect_label").fit(train_data=df_train, time_limit=14400, hyperparameters="multimodal")`

But I keep on getting the TextPredictor error that  "No GPU is detected in the machine".

How do I enable the GPU usage? 
(I can use another instance if this is the problem?)

My code:

```
! pip install -U pip==21.3.1
! pip install -U setuptools wheel
! pip install -U "mxnet<2.0.0"
! pip install -U autogluon

import pandas as pd

import warnings
warnings.filterwarnings("ignore")
  
train_data = pd.read_csv('training.csv')
test_data = pd.read_csv('testing.csv')

from autogluon.tabular import TabularDataset, TabularPredictor

train_data = TabularDataset('training.csv')
set_path = 'model_path'

predictor = TabularPredictor(label="detect_label").fit(train_data=df_train, time_limit=14400, hyperparameters="multimodal")

```
Thanks

**Comment by Innixma** [comment]
This should be fixed in v0.4 release (released on March 9th 2022). Please let us know if you still are facing issues!

**Comment by Innixma** [comment]
Do note that v0.4 is only available on Python 3.7 - 3.9, so the `conda_mxnet_p36` kernel will not work. You can instead use a `p37` kernel. AutoGluon v0.4 does not depend on MXNet anymore, so no need to select a mxnet kernel.
